CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_unified_rag v0.1.0-a3
```

Comment:
Can you add the new alpha version to the README section detailing the install steps.

Reply:
Added!